### ISSUE TITLE
Mathematical/scientific formula display

### ISSUE THREAD

**Issue by StephenOTT** [wet-boew/wet-boew-drupal#279]
Reminder for Mathematical/scientific formula display capabilities
http://wet-boew.github.com/wet-boew/demos/mathlib/mathlib-eng.html

**Comment by sylus** [comment]
I am not sure what type of integration you would like with this? We are using WET's JS so we do have the capability to render this when the markup is correct. For example if you copied the markup from the above link into our tinymce editor the equations would render.

Are you referring to tinymce integration or the sort so equations can be dynamically generated? Likely this feature would be a whole project on its own.

@mgifford can you offer any feedback? I think this feature request might be out of scope.

**Comment by mgifford** [comment]
I wish that the #WET demos had dates on them.  @pjackson28 I'm assuming that the Mathlib link above wasn't actually published 8 days in the future - Date modified: 2012-09-17 

We can't rely (yet) on http://caniuse.com/mathml

The bigger discussion I think is still http://ircan-rican.gc.ca/projects/gcwwwformul but actually I think it's mostly been done through email and not archived.

There were performance issues with IE.  

However, I don't think this is something that just comes with WxT.  It's a package that would need to be added to the Drupal Distro when the problem's been sufficiently fixed in Wxt.

**Comment by StephenOTT** [comment]
Who would be the heaviest user?

**Comment by StephenOTT** [comment]
@sylus 
Based on a future of more content being built directly in drupal would see the integration with a visual formula builder.  Most likely its own app/project/module in the future

**Comment by mgifford** [comment]
I am going to close this issue again. I didn't mean to re-open it.  It doesn't mean that the conversation can't continue, but it really does have to wait for the WET folks.  In Drupal you could just postpone an issue, but that's not an option here.

**Comment by pjackson28** [comment]
@mgifford We set the Date modified values to the WET v3.0 release date in anticipation of the formal release next week. We've also sent Mathematical/scientific formula display for testing. We were able to improve the polyfill detection to correctly identify inadequate MathML support (Safari, Opera) and added known issues regarding IE performance to the working examples.

Note that IE performance is only really an issue on pages with a lot of complex formulas being loaded on slow machines with IE8. IE is sluggish but not too bad on low to medium complexity pages, when using IE7 or IE9, or when using a quick machine.
